src/app/(proper_react)/redesign/(authenticated)/user/dashboard/page.tsx

```diff
@@ -25,14 +26,12 @@ export default async function DashboardPage() {
     return redirect("/user/welcome/");
   }
 
-  return (
-    <div className={styles.wrapper}>
-      <Toolbar session={session}>
-        TODO:{" "}
-        <a href="https://react-spectrum.adobe.com/react-aria/useTabList.html">
-          add a tab list
-        </a>
-      </Toolbar>
-    </div>
-  );
+  if (!session?.user) {
+    return redirect("/");
+  }
```

Comment:
question: Did we have this logic somewhere else?! It seems super important. 😅

Reply:
Oh yeah, I guess technically it's redundant with this: https://github.com/mozilla/blurts-server/blob/8c7dc84860ac780fca25cfe4aa2bfefc50f16f4a/src/app/(proper_react)/redesign/(authenticated)/layout.tsx#L13-L15

Although better safe than sorry I suppose, and now TS also knows for sure that `session.user` is defined.